lmfdb/classical_modular_forms/templates/cmf_browse.html

```diff
@@ -41,7 +41,7 @@ <h2> Browse {{ KNOWL('cmf.newform','newforms')}}</h2>
 {% endfor %}
 
 <p>
-  A <a href="?search_type=Dimensions&dim=1">table</a> of newforms with rational {{ KNOWL('cmf.q-expansion', title='\(q\)-expansion') }}
+  A <a href="?search_type=Dimensions&dim=1">table</a> of newforms with rational {{title}} {{ KNOWL('cmf.q-expansion', title='\(q\)-expansion') }}
```

Comment:
Was this intentional? It looks like maybe it was something I did when experimenting rather than an actual change?